inventory/hosts.example

```diff
@@ -838,8 +838,12 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # interface other than the default network interface.
 #openshift_set_node_ip=True
 
-# Configure dnsIP in the node config
-#openshift_dns_ip=172.30.0.1
+# Configure dnsIP in the node config. Set openshift_dns_ip per node as needed in
```

Comment:
I think we should add something about this not being needed in most cases and try to define the default behavior when we don't set this.